### DISCUSSION TITLE
Error: Call retries were exceeded

### DISCUSSION THREAD

**Discussion by yeabsra-leul** [vercel/next.js#47103] [Help]
### Summary

when I try to implement dynamic routes i get " Error: Call retries were exceeded "

**Comment by icyJoseph** [comment]
Looking at the code, I see that, perhaps, this happens whenever there's some type error in your system. 

- test/production/ci-missing-typescript-deps/index.test.ts, has a test where this string is expected to not be present
- packages/next/src/build/index.ts, here's were the check for that particular message is made

More specifically, it comes from the worker pool used by Next.js to build your app. 

I think I'd need to see a reproduction demo of your issue to be able to help you. To be precise this issue is created inside a `jest-worker` which is the same type of worker used by the `jest` runtime, but since it is "just a worker", it is used inside Next.js to build pages.

- https://github.com/facebook/jest/blob/main/packages/jest-worker/src/workers/NodeThreadsWorker.ts#L144
